.circleci/config.yml

```diff
@@ -1,178 +1,421 @@
-# Ruby CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-ruby/ for more details
-#
 version: 2.1
-parameters:
-  ruby-version:
-    type: string
-    default: "3.0.3"
-  postgres-version:
-    type: string
-    default: "11.13"
-executors:
-  docker-publisher:
-    environment:
-      IMAGE_NAME: suldlss/suri-rails
-    docker:
-    - image: cimg/base
-jobs:
-  test:
-    docker:
-      - image: 'cimg/ruby:<< pipeline.parameters.ruby-version >>-node'
+orbs:
+  ruby-rails:
+    orbs:
+      docker: circleci/docker@2.0.1
+      node: circleci/node@3.0
+      ruby: circleci/ruby@1.2.0
+      browser-tools: circleci/browser-tools@1.2.3
+    references:
+      ruby-version-ref: &ruby-version-ref '2.7.5'
+      postgres-version-ref: &postgres-version-ref '11.13'
+      ruby-executor-refs: &ruby-executor-refs
+        image: 'cimg/ruby:<< parameters.ruby-tag >>-node'
```

Comment:
not convinced suri-rails needs node??   Might be good to avoid node when we don't need it?

Reply:
suri-rails shouldn't need node. However, there is no harm in having node installed in the container used for building.